vite.config.js

```diff
@@ -10,7 +10,13 @@ export default defineConfig({
       name: 'ScrollTimeline',
       // the proper extensions will be added
       fileName: (format, entryAlias) => `scroll-timeline${format=='iife'?'':'-' + format}.js`,
-      formats: ['iife']
+      formats: ['iife'],
+    },
+    minify: 'terser',
+    terserOptions: {
+      mangle: {
+        keep_classnames: /(Scroll|View)Timeline/
```

Comment:
Might even be good to just keep all classnames since as you pointed out in the other PR that some of the CSSOM names are important, it barely affects the size since there's not actually that many classes defined in the polyfill anyways.